velox/row/UnsafeRowFast.cpp

```diff
@@ -27,12 +27,155 @@ int32_t alignBits(int32_t numBits) {
 int32_t alignBytes(int32_t numBytes) {
   return bits::roundUp(numBytes, 8);
 }
+
+int32_t firstNonzeroLongNum(const std::vector<int32_t>& mag, int32_t length) {
```

Comment:
naming: firstNonzeroLongNum -> firstNonZeroLongNum

Would you add a comment to explain what this method does?

naming: do not abbreviate: is there a more descriptive name for 'mag' argument?

Reply:
It is `magnitude`, to align with java BigInteger, it is `mag`